src/index.ts

```diff
@@ -38,14 +39,20 @@ async function main() {
   try {
     console.log('Started refreshing application (/) commands.');
 
-    await rest.put(Routes.applicationGuildCommands(clientID!, guildID!), { body: [Ping.info.toJSON()] });
+    await rest.put(Routes.applicationGuildCommands(clientID!, guildID!), { body: [
+      Ping.info.toJSON(),
+      Mock.info.toJSON()
+      ] 
+    });
 
     console.log('Successfully reloaded application (/) commands.');
   } catch (error) {
     console.error(error);
   }
 }
 
+export let mockTargets = new Set();
```

Comment:
We could use `const` instead of `let` here. `const` means they can't redefine `mockTargets`, but they can still modify the "contents" of `mockTarget`.